console/src/components/Common/utils/v1QueryUtils.ts

```diff
@@ -98,6 +98,65 @@ export const getDropPermissionQuery = (
   };
 };
 
+export const getInsertUpQuery = (
+  tableDef: TableDefinition,
+  insertion: object
+) => {
+  return {
+    type: 'insert',
+    args: {
+      table: tableDef,
+      returning: [],
+      object: insertion,
+    },
+  };
+};
+
+interface ColumnType {
+  column_name: string;
+}
+
+type PrimaryKeyInfo = {
+  table_name: string;
+  table_schema: string;
+  constraint_name: string;
+  columns: Array<string>;
+};
+
+export const getInsertDownQuery = (
+  tableDef: TableDefinition,
+  insertion: Record<string, any>,
+  primaryKeyInfo: PrimaryKeyInfo,
+  columns: Array<ColumnType>
+) => {
+  const whereClause: any = {};
+
+  const hasPrimaryKeys = primaryKeyInfo?.columns;
```

Comment:
Since it's duplicated code we can move this to `pgUtils` as, for example, `createPKClause`.